mbed_build/_internal/templates/mbed_config.tmpl

```diff
@@ -1,28 +1,63 @@
-/*
-* Copyright (C) 2020 Arm Mbed. All rights reserved.
-* SPDX-License-Identifier: Apache-2.0
-*/
+# Copyright (C) 2020 Arm Mbed. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
 
-// Automatically generated configuration file.
-// DO NOT EDIT, content will be overwritten.
+# Automatically generated configuration file.
+# DO NOT EDIT, content will be overwritten.
```

Comment:
Replace `,` with `.` and capitalize `c` as `C`. Maybe also prefer "may" to "will", since maybe it won't always be overwritten, for example if someone doesn't run the command.

Reply:
Yeah. This was a copy and paste job and I didn't do the necessary grammar check.